<!DOCTYPE html>
<html lang='en'>

<head>
    <base href=".">
    <link rel="shortcut icon" type="image/png" href="assets/favicon.png"/>
    <link rel="stylesheet" type="text/css" media="all" href="assets/main.css"/>
    <meta name="description" content="Conference Template">
    <meta name="resource-type" content="document">
    <meta name="distribution" content="global">
    <meta name="KeyWords" content="Conference">
    <title>MLMG-2022</title>
</head>

<body>

    <div class="banner">
        <img src="assets/banner_ecml_2.jpg" alt="Conference Template Banner"
        width=1000>
        <div class="top-left">
            <span class="title1" style="font-size:40.0pt">ML</span><span class="title2" style="font-size:40.0pt">Microbial Genomics</span> <span class="year">2022</span>
        </div>
        <div class="bottom-right">
            <span style="font-size:25.0pt"> September 23rd, 2022 - Virtual Event </span>
        </div>
    </div>

    <table class="navigation">
        <tr>
            <td class="navigation">
                <a class="current" title="Conference Home Page" href=".">Home</a>
            </td>
            <td class="navigation">
                <a title="Call for Papers" href="call">Call for papers</a>
            </td>
            <td class="navigation">
                <a title="Conference Program" href="program">Program</a> 
            </td>
            <td class="navigation">
                <a title="Selected abstracts" href="abstracts">Abstracts</a> 
            </td>
            <td class="navigation">
                <a title="Practical information" href="practical">Practical information</a>
            </td>
        </tr>
    </table>

    <h2>Machine Learning for Microbial Genomics 2022</h2>

    <p>
        Next-generation sequencing (NGS) technologies are getting cheaper and cheaper, making it
possible to sequence massive panels of microbial genomes. Accordingly, NGS is becoming a
key technological pillar to understand and fight microbial infections. This was exemplified in the
context of the on-going COVID-19 pandemic, where NGS technologies offered the possibility to
identify novel variants of the SARS-CoV2 virus, and to monitor in real-time their worldwide
evolution. Likewise, they are increasingly used to decipher the genetic bases of antimicrobial
resistance (AMR), which is among the most important public health concerns. They also allow
us to characterize with an unprecedented level of resolution the diversity of microbial flora,
making it possible to investigate the relationship between microbiomes and various diseases,
thereby opening new avenues in terms of diagnostics and therapies.
    </p>

    <p>
The improvement and increasing accessibility of NGS data call for innovative data analysis
strategies to extract meaningful and actionable information. Machine learning approaches are
more and more popular in this context, offering alternative and complementary tools beyond
bioinformatics to analyze genomic data. Our objective is to gather a community of researchers
working at the interface of machine learning and computational biology to study microbial
genomic data.
    </p>

    <p>
Typical problems of interest include:
<ul>
<li>Improving microbial genome-wide association studies (GWAS), e.g. by proposing more
relevant representations of genetic variation, novel testing procedures or better ways to
deal with the strong population structure encountered in typical bacterial panels.</li>
<li>Predicting a phenotype of interest (e.g. AMR, hypervirulence) from a microbial genome.</li>
<li>Inferring multiple aspects of a populationâ€™s evolutionary history from a set of microbial
genomes, e.g. the presence of selective sweeps and their date, expansions and declines
linked to environmental changes or public health policies, recombination across multiple
host species, and its phylogenetic tree.</li>
<li>Improving our ability to visualize microbial genomes in a way that highlights relevant
elements with respect to a phenotype of interest.</li>
<li>Estimating the constitution of a microbial flora, its diversity and (un)-stability and
evaluating its association with a phenotypic trait of the host (e.g. obesity, inflammatory
bowel disease).</li>
</ul>
</p>


<h2>Keynote speakers</h2>


    <table class="po">
        <tr>
            <td><img src="assets/Alice_McHardy.jpg" height="220"></td>
            <td><img src="assets/wheeler-nicole-2021.jpg" height="250"></td>
        </tr>
        <tr>
            <td>
                <a href="https://www.helmholtz-hzi.de/en/research/research-topics/bacterial-and-viral-pathogens/computational-biology-of-infection-research/alice-mchardy/">Alice McHardy</a>
            </td>
            <td>
                <a href="https://www.birmingham.ac.uk/staff/profiles/microbiology-infection/wheeler-nicole.aspx">Nicole Wheeler</a>
            </td>
        </tr>
    </table>


<h2>Organizers</h2>
    <p>
        Meriem El Azami, Laurent Jacob, Flora Jay, Pierre Mah&eacute; :  <a HREF="mailto:mlmg2022@lisn.upsaclay.fr">contact</a>
    </p>

    <footer>
        &copy; Conference Organizers
        &nbsp;|&nbsp; Design by <a href="https://github.com/mikepierce">Mike Pierce</a>
    </footer>

</body>
</html>

