<!DOCTYPE html>
<html lang='en'>

<head>
    <base href="..">
    <link rel="shortcut icon" type="image/png" href="assets/favicon.png"/>
    <link rel="stylesheet" type="text/css" media="all" href="assets/main.css"/>
    <meta name="description" content="Conference Template">
    <meta name="resource-type" content="document">
    <meta name="distribution" content="global">
    <meta name="KeyWords" content="Conference">
    <title>MLMG-2022</title>
</head>

<body>

    <div class="banner">
        <img src="assets/banner_ecml_2.jpg" alt="Conference Template Banner"
        width=1000>
        <div class="top-left">
            <span class="title1" style="font-size:40.0pt">ML</span><span class="title2" style="font-size:40.0pt">Microbial Genomics</span> <span class="year">2022</span>
        </div>
        <div class="bottom-right">
            <span style="font-size:25.0pt"> September 23rd, 2022 - Virtual Event </span>
        </div>
    </div>

    <table class="navigation">
        <tr>
            <td class="navigation">
                <a title="Conference Home Page" href=".">Home</a>
            </td>
            <td class="navigation">
                <a class="current" title="Call for Abstracts" href="call">Call for Abstracts</a>
            </td>
            <td class="navigation">
                <a title="Conference Program" href="program">Program</a> 
            </td>
            <td class="navigation">
                <a title="Selected abstracts" href="abstracts">Abstracts</a> 
            </td>
            <td class="navigation">
                <a title="Practical information" href="practical">Practical information</a>
            </td>
        </tr>
    </table>

    <br>

    <h2>Key dates</h2>
    <ul>
      <li>Submission Deadline: <del>June 20th 2022</del> <ins>June 27th 2022</ins></li>
      <li>Decision: <del>July 13th 2022</del> <ins>July 20th 2022</ins></li>
    </ul> 
    
    <h2>Call for paper</h2>



<p>
    We invite submissions to the ECML-PKDD 2022 workshop on machine learning for microbial
genomics to be held virtually, on September 23rd.
</p> 

<p>
    The improvement and increasing
accessibility of Next Generation Sequencing data calls for innovative data analysis strategies to
extract meaningful and actionable information. Machine learning approaches are more and
more popular in this context, offering alternative and complementary tools beyond bioinformatics
to analyze genomic data. However, despite the recent advances, their application remains
challenging and new approaches that are robust, scalable to large-scale high-dimensional data
and interpretable are needed.
</p>

<p>
    This workshop aims to bring together interdisciplinary researchers working at the interface of
machine learning and computational biology for the study of microbial genomic data to discuss
recent advances in this field. We will also have the privilege of hosting two distinguished speakers from this community to
discuss and present their current research.
</p> 

<p>Typical topics of interest include:
    <ul>
        <li>Improving microbial genome wide association studies (GWAS).</li>
        <li>Phenotype prediction from microbial genomes.</li>
        <li>Inferring population parameters from a set of microbial genomes.</li>
        <li>Visualization of microbial genomes in a way that highlights relevant elements with respect to a phenotype of interest.</li></li>
        <li>Study of the constitution of a microbial flora.</li>
    </ul>
</p>

    <h2>Instructions to authors</h2>


<p>
 
    Papers must be written in English and formatted according to the <a href="https://www.springer.com/gp/computer-science/lncs/conference-proceedings-guidelines"
    target="_blank" rel="noopener noreferrer"> Springer LNCS template</a>.
    
    <br><br>

    Authors must submit their manuscript using the <a href="https://easychair.org/conferences/?conf=mlmg2022"
    target="_blank" rel="noopener noreferrer">EasyChair platform</a>.

    <br><br>
    Submissions may not exceed 6 pages in PDF format, including figures and references (which roughly corresponds to 3 pages with standard margins).

    <br><br>
    Submitted papers will be (single blind) peer reviewed and selected papers will be presented in 20 minute talks.
    <br>
    The accepted papers will be published on the workshop’s website, but will not be considered as conference proceedings.

    <br><br>
    Please note that at least one author of each accepted paper has to register for the main conference
 
</p>


    <h2>Program Committee</h2>
<p>
    <ul>
       <li>Franz Baumdicker (University of Tübingen)</li>
       <li>Julien Chiquet (University Paris-Saclay, AgroParisTech, INRAE)</li>
       <li>Lauren Cowley (University of Bath)</li>
       <li>Adrian Egli (University Hospital Basel)</li>
       <li>Maha Farhat (Harvard)</li>
       <li>Bogdan Iorga (ICSN-CNRS)</li>
       <li>John Lees (EMBL-EBI)</li>
       <li>Mahendra Mariadassou (INRAE)</li>
       <li>Macha Nikolski (University of Bordeaux)</li>
       <li>Jean-Philippe Rasigade (Hospices Civils de Lyon, University of Lyon)</li>
       <li>Simon Rasmussen (University of Copenhagen)</li>
       <li>Jean-Philippe Vert (Google Brain)</li>
       <li>Daniel Wilson (University of Oxford)</li>
    </ul>

</p>



    <footer>
        &copy; Conference Organizers
        &nbsp;|&nbsp; Design by <a href="https://github.com/mikepierce">Mike Pierce</a>
    </footer>

</body>
</html>

